src/Framework/TestCase.php

```diff
@@ -544,6 +544,33 @@ protected function checkRequirements()
     }
 
     /**
+     * Skips the current test if --percentage-skip was used and the current PHPUnit test run is not the required
+     * percentage complete.
+     */
+    protected function checkPercentageSkip()
```

Comment:
Please make this `private`. By default, new non-public methods should be private. We can always open it up later to `protected` if need be but we cannot close it down from `protected` to `private`.